applications/dashboard/scss/editor/_embed.scss

```diff
@@ -264,6 +265,15 @@ $embed-focus-color: #0291DB;
     width: $embedSoundCloud_width;
 }
 
+// Getty
+
+.embed-getty {
+    width: 100%;
+    max-width: $embedGetty_width;
+}
+embedExternal-content.embed-getty{
```

Comment:
This is not a good CSS-selector at all. I'd just remove it.

For reference this will lookup the following:

```html
<embedExternal-content class="embed-getty"></embedExternal-content>
```